README.md

```diff
@@ -1,6 +1,6 @@
 # MetaMask Browser Extension
 
-You can find the latest version of MetaMask on [our official website](https://metamask.io/). For help using MetaMask, visit our [User Support Site](https://metamask.zendesk.com/hc/en-us).
+You can find the latest version of MetaMask on [our official website](https://metamask.io). For help using MetaMask, visit our [User Support Site](https://metamask.zendesk.com/hc/en-us).
```

Comment:
Hi @rahulk2109, were you having trouble accessing this URL? This link appears to work for me with the slash.